File: Modul_6/PRAK605-2510817220010-AuroraShafaSalsabila.c
#include <stdio.h>

int main(){
    int ordo;
    scanf("%d", &ordo);
    int matriks_A[ordo][ordo], matriks_B[ordo][ordo], matriks_AXB[ordo][ordo];
    printf("Matriks A \n");
    
    for(int i = 0; i < ordo; i++){
        for( int j = 0; j < ordo; j++){
            scanf("%d", &matriks_A[i][j]);
        }
    }

    printf("Matriks B \n");

    for(int i = 0; i < ordo; i++){
        for( int j = 0; j < ordo; j++){
            scanf("%d", &matriks_B[i][j]);
        }
    }
 
    printf("Matriks AXB \n", ordo, ordo);

    for(int i = 0; i < ordo; i++){
        for(int j = 0; j < ordo; j++){
            matriks_AXB[i][j] = 0;
            for (int k = 0; k < ordo; k++){
                matriks_AXB[i][j] += matriks_A[i][k] * matriks_B[k][j];
            }
        }
    }

    for(int i = 0; i < ordo; i++){
        for(int j = 0; j < ordo; j++){
            printf("%d ", matriks_AXB[i][j]);
        }
        printf("\n");
    }

    return 0;
}

File: Modul_6/PRAK605-2510817220010-AuroraShafaSalsabila.py
ordo = int(input())

print("Matriks A")
matriks_A = []
for i in range(ordo):
    matriks_A.append(list(map(int, input().split())))

print("Matriks B")
matriks_B = []
for i in range(ordo):
    matriks_B.append(list(map(int, input().split())))

matriks_AxB = [[0 for _ in range(ordo)] for _ in range(ordo)]

for i in range(ordo):
    for j in range(ordo):
        for k in range(ordo):
            matriks_AxB[i][j] += matriks_A[i][k] * matriks_B[k][j]

print(f"Matriks AXB")
for row in matriks_AxB:
    print(*row)